.github/workflows/deploy_gh_pages.yml

```diff
@@ -2,6 +2,11 @@ name: Deploy Docs to GitHub Pages
 
 on:
   workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - 'site/**'
```

Comment:
Is it possible to deploy it automatically to contributor's github.io? That way this workflow could be triggered on pull_request automatically as well

Reply:
Discussed. Agreed to keep manual preview deployment for now (from fork) and investigate possible automation options.